lib/matplotlib/collections.py

```diff
@@ -1132,7 +1132,7 @@ def legend_elements(self, prop="colors", num="auto",
 
         kw = dict(markeredgewidth=self.get_linewidths()[0],
                   alpha=self.get_alpha())
-        kw.update(kwargs)
+        kw.update(**kwargs)
```

Comment:
Why this change? Both should work, but the original one is easier to read.

Reply:
sorry, reverted, that was just a bad search/replace.